build.gradle

```diff
@@ -98,22 +99,45 @@ configure(javaProjects) {
             dependency("org.opensaml:opensaml:2.6.4")
             dependency("org.python:jython-standalone:2.7.1")
             dependency("org.springframework.security.extensions:spring-security-saml2-core:1.0.2.RELEASE")
+            dependencySet(group: "com.google.protobuf", version: "3.4.0") {
+              entry "protobuf-java"
+              entry "protoc"
+            }
+            dependencySet(group: "io.grpc", version: "1.6.1") {
+              entry "grpc-core"
+              entry "grpc-netty"
+              entry "grpc-stub"
+              entry "grpc-protobuf"
+              entry "protoc-gen-grpc-java"
+              entry "grpc-testing"
+            }
+            dependencySet(group: "net.devh", version: "1.3.0-RELEASE") {
+              entry "grpc-server-spring-boot-autoconfigure"
+              entry "grpc-server-spring-boot-starter"
+            }
         }
     }
 
     ext {
         generatedSourcesDir = file("${buildDir}/generated-sources/main")
         generatedSourcesJavaDir = file(new File(generatedSourcesDir, "/java"))
+        generatedSourcesGRPCDir = file("${buildDir}/generated-grpc/main/java")
```

Comment:
Should this not extend from the `generatedSourcesDir` the same way we do with Java? Or why not just dump it in to the same java generated sources dir?

Reply:
This is a limitation of the gradle protobuf plugin, I don't remember the details because it was before the break, but I spent a day or two making this work with generated files in the same folder and there's no way to.
It has to do with hardcoded path components that the gradle protobuf plugin appends. In the end it would end up in `main/main/java` or something like that